easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb

```diff
@@ -0,0 +1,63 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'Java'
+easyblock = 'ConfigureMake'
```

Comment:
@edmondac Style nitpicking: this should go up, above `name =`, with an empty line in between.